### ISSUE TITLE
[Feature] Alternative MultiDataModel instantiation with image_uri instead of model

### ISSUE THREAD

**Issue by austinmw** [aws-samples/sagemaker-ssh-helper#7]
Hi, thanks for developing this repo. Is it currently possible, or would it be possible to add capability, to connect to an inference endpoint when [MultiDataModel](https://sagemaker.readthedocs.io/en/stable/api/inference/multi_data_model.html#sagemaker.multidatamodel.MultiDataModel) is instantiated using the alternative method of providing an `image_uri` instead of a `model`? For example:

```python
endpoint_name = "openmmlab-mms-" + datetime.now().strftime("%Y-%m-%d-%H-%M-%S")

multi_model_s3uri = f"s3://{bucket}/openmmlab-mms/"

mme = MultiDataModel(
    name=endpoint_name,
    image_uri=image_uri,
    model_data_prefix=multi_model_s3uri,
    sagemaker_session=session,
    role=role,
)

predictor = Predictor(endpoint_name, sagemaker_session=session)
predictor.serializer = sagemaker.serializers.IdentitySerializer()
predictor.deserializer = sagemaker.deserializers.JSONDeserializer()

mme.add_model(model_data_source=estimator.model_data, model_data_path='modelA')
```
----

### Additional context

The reason I am using MME in this way instead of providing an initial model, is because I'm using the MMDetection library's [mmdet2torchserve.py](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/tools/deployment/mmdet2torchserve.py) tool. This results in my training job producing a `model.tar.gz` with the following four files:

```
MAR-INF/MANIFEST.json
mmdet_handler.py
config.py
bbox-mAP_epoch_12.pth
```

Where `mmdet_handler.py` is defined [here](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/tools/deployment/mmdet_handler.py) and does not implement SageMaker's standard `input_fn`, `predict_fn` and `output_fn`. Ideally I would like to be able to use this export as-is without needing to download, extract and refactor the handler script in order to provide an `entry_point` (and inference does indeed appear to work correctly when I do this).

**Comment by ivan-khvostishkov** [comment]
Hi, Austin. Thank you for asking! Currently it's not possible to pass the arguments 'image_uri', 'role' and 'kwargs' into MultiDataModel and create an SSHMultiModelWrapper for this model. Let's keep this issue open as a feature request, I'll consider adding this feature in the future.

**Comment by ivan-khvostishkov** [comment]
Hi, @austinmw . The support of MultiDataModel without passing the model object was released in v1.9.0.

Check the corresponding [section of the FAQ](https://github.com/aws-samples/sagemaker-ssh-helper/blob/main/FAQ.md#what-if-i-want-to-deploy-a-multi-data-model-without-passing-a-reference-to-a-model-object-only-with-image_uri): "What if I want to deploy a Multi Data Model without passing a reference to a Model object, only with image_uri?".
